features/events/traps/src/test/resources/blueprint-empty-camel-context.xml

```diff
@@ -0,0 +1,18 @@
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
```

Comment:
Get rid of this file, you can include ```blueprint-empty-camel-context.xml``` from the classpath now because it is included in the ```org.opennms.core.test-api.camel``` project.